### PR TITLE
Feat/gnome remote desktop

### PR DESCRIPTION
**Pull request by tarkah**

### CODE REVIEW

File: l/libxv/stone.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    libXv provides an X Window System client interface to the XVideo extension to the X protocol.
+license    : MIT
+emul32     : yes
+builddeps  :
+    - pkgconfig32(xext)
+    - pkgconfig32(x11)
+    - pkgconfig(xproto)
+setup      : |
+    %configure \
+        --prefix=/usr \
```
Comment (joebonrichie) [inline l/libxv/stone.yaml:23]: nit: unnecessary part of macro

**Review by joebonrichie — CHANGES_REQUESTED** [review]
one small nit

File: l/libxv/stone.yaml (hunk 1 above)
Comment (tarkah) [reply l/libxv/stone.yaml]: good catch, fixed :+1:

File: f/ffnvcodec-headers/stone.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name       : ffnvcodec-headers
```
Comment (ReillyBrogan) [inline f/ffnvcodec-headers/stone.yaml:6]: I have a slight preference for naming this package `nv-codec-headers`

File: g/gnome-remote-desktop/stone.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+builddeps   :
+    - binary(mutter)
+    - pkgconfig(cairo)
+    - pkgconfig(dbus-1)
+    - pkgconfig(epoxy)
+    - pkgconfig(fdk-aac)
+    - pkgconfig(ffnvcodec)
+    - pkgconfig(freerdp3)
+    - pkgconfig(freerdp-client3)
+    - pkgconfig(freerdp-server3)
```
Comment (ReillyBrogan) [inline g/gnome-remote-desktop/stone.yaml:26]: These are all provided by the same devel package, you don't need all of them.

File: g/gnome-remote-desktop/stone.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    - pkgconfig(libnotify)
+    - pkgconfig(libpipewire-0.3)
+    - pkgconfig(libsecret-1)
+    - pkgconfig(opus)
+    - pkgconfig(polkit-gobject-1)
+    - pkgconfig(systemd)
+    - pkgconfig(tss2-esys)
+    - pkgconfig(tss2-mu)
+    - pkgconfig(tss2-rc)
+    - pkgconfig(tss2-tctildr)
```
Comment (ReillyBrogan) [inline g/gnome-remote-desktop/stone.yaml:44]: Same here.

File: l/libxv/stone.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+version    : '1.0.12'
+release    : 1
+homepage   : https://www.x.org/
+upstreams  :
+    - https://www.x.org/releases/individual/lib/libXv-1.0.12.tar.gz : ce706619a970a580a0e35e9b5c98bdd2af243ac6494c65f44608a89a86100126
+summary    : X11 Video extension library
+description: |
+    libXv provides an X Window System client interface to the XVideo extension to the X protocol.
+license    : MIT
+emul32     : yes
```
Comment (ReillyBrogan) [inline l/libxv/stone.yaml:16]: This looks left over from whatever package you copied and pasted as your base. I doubt we need the 32bit libraries for this (at least not yet)

File: o/opus/stone.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+version    : '1.5.2'
+release    : 1
+homepage   : https://www.opus-codec.org/
+upstreams     :
+    - https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz : 65c1d2f78b9f2fb20082c38cbe47c951ad5839345876e46941612ee87f9a7ce1
+summary    : Opus is a totally open, royalty-free, highly versatile audio codec.
+description: |
+    Opus is a totally open, royalty-free, highly versatile audio codec. Opus is unmatched for interactive speech and music transmission over the Internet, but is also intended for storage and streaming applications. It is standardized by the Internet Engineering Task Force (IETF) as RFC 6716 which incorporated technology from Skype’s SILK codec and Xiph.Org’s CELT codec.
+license    : BSD-3-Clause
+emul32     : yes
```
Comment (ReillyBrogan) [inline o/opus/stone.yaml:16]: Same here re emul32

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
We also should have monitoring.yaml files for all of these. You can copy and paste the Solus ones if they exist, and if they don't we can add them to Solus as well.

File: g/gnome-remote-desktop/stone.yaml (hunk 3 above)
Comment (tarkah) [reply g/gnome-remote-desktop/stone.yaml:26]: For sure, these are distinct dependencies looked up from the Meson side. I figured it was best practice to list them separately on the off chance they were provided by different packages in the future (most likely never a chance here)

**Comment by tarkah** [comment]
> We also should have monitoring.yaml files for all of these. You can copy and paste the Solus ones if they exist, and if they don't we can add them to Solus as well.

Added :+1:

File: o/opus/stone.yaml (hunk 6 above)
Comment (tarkah) [reply o/opus/stone.yaml]: Fixed

File: l/libxv/stone.yaml (hunk 5 above)
Comment (tarkah) [reply l/libxv/stone.yaml]: Fixed

File: g/gnome-remote-desktop/stone.yaml (hunk 4 above)
Comment (tarkah) [reply g/gnome-remote-desktop/stone.yaml:44]: See https://github.com/serpent-os/recipes/pull/328#discussion_r1722636002

File: f/ffnvcodec-headers/stone.yaml (hunk 2 above)
Comment (tarkah) [reply f/ffnvcodec-headers/stone.yaml]: Fixed

File: f/freerdp/stone.yaml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    - pkgconfig(libcjson)
+    - pkgconfig(libpulse)
+    - pkgconfig(libsystemd)
+    - pkgconfig(libusb-1.0)
+    - pkgconfig(libxslt)
+    - pkgconfig(mit-krb5)
+    - pkgconfig(openssl)
+    - pkgconfig(opus)
+    - pkgconfig(sdl2)
+    - pkgconfig(SDL2_ttf)
```
[the remark this answers is not in the document]
Comment (tarkah) [reply f/freerdp/stone.yaml]: Fixed

**Review by ikeycode — APPROVED** [review]
go go gadget build

**Pull request merged**